src/scaffold.js

```diff
@@ -0,0 +1,192 @@
+const fs = require('fs-extra')
+const path = require('path')
+const prettier = require('prettier')
+const { getSubgraphBasename } = require('./command-helpers/subgraph')
+const { step } = require('./command-helpers/spinner')
+const { ascTypeForEthereum, valueTypeForAsc } = require('./codegen/types')
+
+const abiEvents = abi => abi.filter(item => item.type === 'event')
+
+// package.json
+
+const generatePackageJson = ({ subgraphName }) =>
+  prettier.format(
+    JSON.stringify({
+      name: getSubgraphBasename(subgraphName),
+      license: 'UNLICENSED',
+      scripts: {
+        codegen: 'graph codegen',
+        build: 'graph build',
+        deploy:
+          `graph deploy ` +
+          `--node https://api.thegraph.com/deploy/ ` +
+          `--ipfs https://api.thegraph.com/ipfs/ ` +
+          subgraphName,
+        'create-local': `graph create --node http://localhost:8020/ ${subgraphName}`,
+        'remove-local': `graph remove --node http://localhost:8020/ ${subgraphName}`,
+        'deploy-local':
+          `graph deploy ` +
+          `--node http://localhost:8020/ ` +
+          `--ipfs http://localhost:5001 ` +
+          subgraphName,
+      },
+      dependencies: {
+        '@graphprotocol/graph-cli': '0.7.2',
```

Comment:
We'll need to be careful to remember to update this, at least on breaking changes.

Reply:
Indeed. I have an idea: How about we take the package version and put it in here? Then we're always up to date.

Reply:
If that's possible that would be great.